scripts/clean-build.sh

```diff
@@ -1,37 +0,0 @@
-  
```

Comment:
I removed this script, because there was an error in https://gitlab.parity.io/parity/nomidot/-/jobs/348609#L210, surely due to bash on linux and macos being not 100% the same.

rimraf is cross-platform